### ISSUE TITLE
Fetch: get response body in case of 5xx response

### ISSUE THREAD

**Issue by vasily-kirichenko** [fable-compiler/fable-powerpack#57]
In case of 5xx response, `exception.Message` contains nothing useful:

![image](https://user-images.githubusercontent.com/873919/47614233-8f1c6400-daad-11e8-8432-34b030bbd361.png)

![image](https://user-images.githubusercontent.com/873919/47614162-3dbfa500-daac-11e8-9c00-baae28bfede1.png)

The server is based on Akka-Http, and handler is as following:
```scala
val route: Route =
    handleExceptions(exceptionHandler) {
        path("account") {
            post {
                entity(as[AccountDto]) { account =>
                complete(...a Future that can fail...)
            }
        }
} ~ ...

val exceptionHandler = ExceptionHandler {
    case e: Exception =>
        extractUri { uri =>
            println(s"Request to $uri could not be handled normally: ${e.getMessage}")
            complete(HttpResponse(StatusCodes.InternalServerError, entity = e.getMessage))
        }
}
```
![image](https://user-images.githubusercontent.com/873919/47614206-2d5bfa00-daad-11e8-8916-6ae8950fdac2.png)

*In short, what I need is get the following response body in case of 5xx response:*

![image](https://user-images.githubusercontent.com/873919/47614214-449ae780-daad-11e8-9ea4-ee91e941c9cc.png)

Is it possible?

**Comment by vasily-kirichenko** [comment]
@Zaid-Ajaj Thank you! I ended up with this function:
```fsharp
module Fetch =
    let fetch url (init: RequestProperties list) : JS.Promise<Response> =
        promise { 
            let! resp = GlobalFetch.fetch(RequestInfo.Url url, requestProps init)

            match resp.Status with 
            | 200 -> return resp
            | 500 -> 
                let! respText = resp.text()
                return! raise (exn (respText)) 
            | _ ->
                let! respText = resp.text() 
                return! raise (exn (sprintf "Http error from server occured while making request to %s: %s" url respText))
        }
```
